examples/mnist_gan.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import os
+import sys
+sys.path.append(os.getcwd())
+sys.path.append(os.path.join(os.getcwd(), 'test'))
+
+from tinygrad.tensor import Tensor, Function, register
+from tinygrad.utils import get_parameters
+import tinygrad.optim as optim
+
+GPU = os.getenv("GPU") is not None
+
+class Dropout(Function):
```

Comment:
@geohot very needed ..

Reply:
This def doesn't have to be a first class op (with a forward and backward). It's just a multiply by a mask.

Reply:
But for backward you need same mask 
How u will generate those mask again to multiply with gradients

Reply:
Oh so u mean making it second class
Like relu

Reply:
These second class are funnn lol